app/src/main/java/org/p2p/wallet/root/RootActivity.kt

```diff
@@ -105,19 +110,24 @@ class RootActivity : BaseMvpActivity<RootContract.View, RootContract.Presenter>(
     @SuppressLint("SetTextI18n")
     private fun initializeDebugDrawer() {
         val devView = findViewById<TextView>(R.id.developTextView)
+        val isDebugHidden = sharedPreferences.getBoolean(KEY_DEBUG_HIDDEN, false)
```

Comment:
```suggestion
        val isDebugPanelHidden = sharedPreferences.getBoolean(KEY_DEBUG_HIDDEN, false)
```

we can't hide debug itsself :)

Reply:
but it's not panel, it's debug view

Reply:
I can rename to `isDebugViewHidden` if you don't mind @gslevinkov